trim.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice! 0...10
+  end
```

Comment:
- [ ] Great job working on the decorate classes but I think you forgot to add the condition as mentioned in the requirements that makes sure that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word. so please add this condition if it is longer than 10 trim the word otherwise return the name 🚀💯✨✅

Reply:
Hi, @Reem-lab please I need your assistance

Reply:
@Reem-lab, I have fixed the code by adding a conditional statement to check the length before deciding whether to trim the name. Please am I in the right track?
Thanks

Reply:
Hello @yayaismaail I think you are good to go! Good job!

Reply:
Hi, @Reem-lab , thanks